### PR TITLE
+flowtype.0.40.0

### PR DESCRIPTION
**Pull request by gabelevi**

### CODE REVIEW

File: packages/flowtype/flowtype.0.40.0/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "James Kyle"
+]
+doc: "http://flowtype.org/docs/getting-started.html"
+license: "BSD3"
+depends: [
+  "base-unix"
+  "base-bytes"
+  "ocamlbuild" {build}
+]
+available: [ocaml-version >= "4.01.0" & ocaml-version < "4.04.0"]
```
Comment (avsm) [inline packages/flowtype/flowtype.0.40.0/opam]: Looks like the minimum version of OCaml needed is now 4.02.0, due to:
```
# File "src/server/findRefs_js.ml", line 40, characters 7-21:
# Error: Unbound value List.sort_uniq
```

Could this constraint please be bumped to `ocaml-version >= "4.02.0"` ?

**Comment by avsm** [comment]
A minor (non-blocking for merge) compatibility note: the repo doesn't currently compile on static musl-libc (e.g. for Alpine Linux) due to an importable version of inotify:

```
# In file included from inotify_stubs.c:41:0:
# inotify_compat.h:9:24: fatal error: asm/unistd.h: No such file or directory
#  #include <asm/unistd.h>
#                         ^
# compilation terminated.
```

I believe the latest inotify bindings have fixed this, if flowtype wants to vendor in a new version.

File: packages/flowtype/flowtype.0.40.0/opam (hunk 1 above)
Comment (avsm) [inline packages/flowtype/flowtype.0.40.0/opam]: Hi @gabelevi -- just wondering if you've had a chance to look at this?

**Comment by gabelevi** [comment]
I've updated the inotify code in master, and it should be in Flow v0.43.0 that we release in two weeks. Thanks for the suggestion!

**Comment by avsm** [comment]
thanks @gabelevi!

**Pull request merged**